File: Module 1 Foundations/1-SimpleCalculator.py


num1=5
num2=3

def add(a,b):
    return a+b
    
def subtract(a,b):
    return a-b
    
def multiply(a,b):
    return a*b
    
def divide(a,b):
    return a/b
    
choices = """Enter your choice:
    1.Add
    2.Subtract
    3.Multiply
    4.Divide
    """
choice = int(input(choices))
match choice:
    case 1:  print(add(num1, num2)) 
    case 2:  print(subtract(num1, num2)) 
    case 3:  print(multiply(num1, num2)) 
    case 4:  print(divide(num1, num2)) 
    case _:  "Invalid Choice"
    

File: Module 1 Foundations/Basics/1-SimpleCalculator.js
let num1 = 5;
num2 = 3;

function add(a, b) {
  return a + b;
}

function subtract(a, b) {
  return a - b;
}

function multiply(a, b) {
  return a * b;
}

function divide(a, b) {
  return a / b;
}

const Choices = `Enter the option:
 1.Add
 2.Sub
 3.Multiply
 4.Divide
 `;
let choice = Number(prompt(Choices));
switch (choice) {
  case 1:
    console.log(add(num1, num2));
    break;
  case 2:
    console.log(subtract(num1, num2));
    break;
  case 3:
    console.log(multiply(num1, num2));
    break;
  case 4:
    console.log(divide(num1, num2));
    break;
  default:
    break;
}
